app/views/listings/new.html.haml

```diff
@@ -0,0 +1,30 @@
+%h1 Create new listing
+
+
+= form_with model: @listing, local: true do |form|
+  %p
+    = form.label :Name
+    %br/
+    = form.text_field :name
+  %p
+    = form.label :Location
+    %br/
+    = form.text_field :location
+  %p
+    = form.label :Description
+    %br/
+    = form.text_area :description  
+  %p
+    = form.label :Availability
+    %br/
+    = form.text_field :availability
+  %p
+    = form.label :Email
+    %br/
+    = form.email_field :email
+  %p
+    = form.label :"Phone number"
+    %br/
+    = form.telephone_field :"phone number"
```

Comment:
should this be `phone_number`?

Reply:
Good idea, then we are in synch between PRs and I think the naming standard makes more sense.

Reply:
Yes, please use a symbol